diagnostic.md

```diff
@@ -36,13 +44,21 @@ Now, we want to get a single donut from the server. Write the `curl` request
 you'd use to retrieve a single donut, using whatever ID you'd like.
 
 ```sh
-# your answer here
+curl --request GET http://10.13.105.206:3000/
 ```
 
 Write an AJAX request to retrieve a single donut from the server.
 
 ```js
-let getDonut = /* your answer here */;
+let getDonut = function(){
+$.ajax({                                    // $ is jquery, exercute ajax in ().  everything inside is an object literal
+url: 'http://10.13.105.206:3000/items/<item#id>',          // a key is url, retrieve an items
+type: 'GET'                                 // tell ajax to get
+})                                          // ajax takes these info and use it
+
+.done(logResponseBody)                      // register that function when you hear done
+.fail(logRequestError);                     // register that function when you hear fail
+};
```

Comment:
This is a correct generic answer. In the case of this specific example, the path should be `http://www.example.com/donuts/23` (or some other number)